metadata-ingestion/src/datahub/cli/check_cli.py

```diff
@@ -15,11 +19,40 @@ def check() -> None:
 
 @check.command()
 @click.argument("json-file", type=click.Path(exists=True, dir_okay=False))
-def mce_file(json_file: str) -> None:
-    """Check the schema of a MCE JSON file."""
+@click.option(
+    "--rewrite",
+    default=False,
+    is_flag=True,
+    help="Rewrite the JSON file to it's canonical form.",
+)
+def metadata_file(json_file: str, rewrite: bool) -> None:
```

Comment:
do you want to leave the old command in for backwards compat, with help text saying: "Deprecated: use metadata_file instead"?

Reply:
Given that we don't reference this command in our docs at all, my suspicion is that nobody was really using it (we really should have telemetry for cli commands by default). We can add the alias + deprecation warning, but I don't think it's necessary.